src/main/java/com/redhat/management/approval/Identity.java

```diff
@@ -0,0 +1,68 @@
+package com.redhat.management.approval;
+
+/**
+ * This class was automatically generated by the data modeler tool.
+ */
+
+public class Identity implements java.io.Serializable {
+
+    static final long serialVersionUID = 1L;
+
+    private java.lang.String account_number;
```

Comment:
can we rename to `accountNumber`?

Reply:
No, these field names will be used as key names for the header encoding later.